src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVM.md

```diff
@@ -160,6 +162,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AvailabilitySetName
```

Comment:
@hyonholee just to clarify, this wasn't added in this PR, but it was missing from the help, correct?

Reply:
Yes.  It was just missing in the help document.